disnake/raw_models.py

```diff
@@ -147,6 +150,62 @@ def __init__(self, data: MessageUpdateEvent) -> None:
             self.guild_id: Optional[int] = None
 
 
+PollEventType = Literal["POLL_VOTE_ADD", "POLL_VOTE_REMOVE"]
+
+
+class RawMessagePollVoteActionEvent(_RawReprMixin):
```

Comment:
This is a pretty lengthy name - thoughts on removing the `Message` part, at the very least?

Reply:
Fine for me, but then we would need to remove them for the related events names too (`on_message_poll_vote_add/remove`)

Reply:
That's fine, I doubt we'll see polls in any context other than messages in the future.